### PR TITLE
Footer and home page

### PR DESCRIPTION
**Pull request by Max2002**

### CODE REVIEW

File: src/sections/Footer/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import "../../assets/styles/variables";
+
+footer {
```
Comment (GCoder-13) [inline src/sections/Footer/styles.module.scss]: ???

Comment (Max2002) [reply src/sections/Footer/styles.module.scss]: In css module cannot be contacted directly selector

File: src/sections/Home/index.js [hunk 2]
```diff
@@ -1,3 +1,12 @@
+import { IconHomeSvg } from '../../assets/icons';
+import st from './styles.module.scss';
+
 export default function Home() {
-  return <div>Home</div>;
+  return (
+    <main className={st.main}>
+      <div className={st.blockSvg}>
+        <IconHomeSvg alt="React logo" />
```
Comment (GCoder-13) [inline src/sections/Home/index.js]: svg not have alt

File: src/sections/Home/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@import "../../assets/styles/variables";
+
+.main {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 2vw;
+  background: url('../../assets/images/mainBack.jpg') no-repeat center center, $white;
+  background-size: cover;
+  flex: auto;
+  //height: 80vh;
```
Comment (GCoder-13) [inline src/sections/Home/styles.module.scss]: delete

File: src/sections/Home/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+@import "../../assets/styles/variables";
+
+.main {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 2vw;
```
Comment (GCoder-13) [inline src/sections/Home/styles.module.scss]: ???

File: src/sections/SignIn/styles.module.scss [hunk 5]
```diff
@@ -72,4 +72,12 @@
   span {
     margin-left: 8px;
   }
+}
+
+.svg {
```
Comment (GCoder-13) [inline src/sections/SignIn/styles.module.scss]: rename

File: src/assets/icons/index.js [hunk 6]
```diff
@@ -10,3 +10,4 @@ export { ReactComponent as DropDownSvg } from './dropdownArrow.svg';
 export { ReactComponent as LogOutIconSvg } from './logoutIcon.svg';
 export { ReactComponent as HomeSvg } from './home.svg';
 export { ReactComponent as ContactsSvg } from './contacts.svg';
+export { ReactComponent as IconHomeSvg } from './iconHomePage.svg';
```
Comment (GCoder-13) [inline src/assets/icons/index.js]: rename

File: src/assets/icons/index.js [hunk 7]
```diff
@@ -10,3 +10,4 @@ export { ReactComponent as DropDownSvg } from './dropdownArrow.svg';
 export { ReactComponent as LogOutIconSvg } from './logoutIcon.svg';
```
Comment (GCoder-13) [inline src/assets/icons/index.js]: LogOutSvg

File: src/sections/Home/styles.module.scss (hunk 4 above)
Comment (Max2002) [reply src/sections/Home/styles.module.scss]: use the same units

File: src/sections/SignIn/styles.module.scss (hunk 5 above)
Comment (Max2002) [reply src/sections/SignIn/styles.module.scss]: i delet this, because don't use in sigIn

File: src/assets/icons/index.js [hunk 8]
```diff
@@ -7,6 +7,7 @@ export { ReactComponent as HidePassSvg } from './hidePass.svg';
 export { ReactComponent as VisiblePassSvg } from './visiblePass.svg';
 export { ReactComponent as IconErrorSvg } from './iconError.svg';
 export { ReactComponent as DropDownSvg } from './dropdownArrow.svg';
-export { ReactComponent as LogOutIconSvg } from './logoutIcon.svg';
+export { ReactComponent as LogOutSvg } from './logout.svg';
 export { ReactComponent as HomeSvg } from './home.svg';
 export { ReactComponent as ContactsSvg } from './contacts.svg';
+export { ReactComponent as SpinReactLogoSvg } from './spinReactLogo.svg';
```
Comment (GCoder-13) [inline src/assets/icons/index.js]: ReactLogoSvg

File: src/sections/Home/index.js [hunk 9]
```diff
@@ -1,3 +1,12 @@
+import { SpinReactLogoSvg } from '../../assets/icons';
+import st from './styles.module.scss';
+
 export default function Home() {
-  return <div>Home</div>;
+  return (
+    <main className={st.main}>
+      <div className={st.blockSvg}>
+        <SpinReactLogoSvg />
```
Comment (GCoder-13) [inline src/sections/Home/index.js]: use classname

**Review by GCoder-13 — APPROVED** [review]

**Pull request merged**